src/library/templates/library/edit_customization.html

```diff
@@ -103,5 +126,146 @@ <h2>Edit {{ customization.title }} {% if is_new == 'true' %}(New){% endif %}</h2
        $input.val($input.val() + text);
        $input.focus();
     });
+
```

Comment:
The JS portion of this should eventually be moved into one of the JS files.

Reply:
Right.  Probably `frontend.js`.  I don't see any JS file specifically for the library pages.  There are some library related scripts in both of `frontend.js` and `toc.js`.